### PR TITLE
UrBackup Plugin

### PR DESCRIPTION
**Pull request by Tostis**
UrBackup Plugin based on official [instructions](https://www.urbackup.org/freenasserverinstall.html).

Artifact repo: https://github.com/Tostis/iocage-plugin-urbackup

### CODE REVIEW

File: urbackup.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "fingerprints": {
+        "plugin-default": [
+            {
+                "function": "sha256",
+                "fingerprint": "b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438"
+            }
+        ]
+    },
+    "official": false,
+    "revision": "0"
```
Comment (sonicaj) [inline urbackup.json:23]: Can we please use an integer value here ( so `0` without the quotes ) ?

File: urbackup.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "UrBackup",
+    "plugin_schema": "2",
```
Comment (sonicaj) [inline urbackup.json:3]: Same here

**Comment by Tostis** [comment]
I've fixed the numbers. I'm sorry for wrong commit date.

File: INDEX [hunk 3]
```diff
@@ -482,6 +482,16 @@
 		"official": false,
 		"primary_pkg": "unifi5"
 	},
+	"urbackup": {
+		"MANIFEST": "urbackup.json",
+		"name": "UrBackup",
+		"icon": "urbackup.png",
```
Comment (sonicaj) [inline INDEX:488]: Can you please change this to `https://icons.freenas.org/community-icons/urbackup.png` as it's supposed to be a URI

**Review by sonicaj — CHANGES_REQUESTED** [review]

File: urbackup.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "UrBackup",
+    "plugin_schema": 2,
```
Comment (fulder) [inline urbackup.json:3]: The `plugin_schema` has been removed from the plugin schema in https://github.com/ix-plugin-hub/iocage-plugin-index/pull/220 so guess we can remove it here now as well (and good to merge the latest `master` into this to see if the manifest looks good according to the latest `iocage` schema.

File: INDEX (hunk 3 above)
Comment (Tostis) [reply INDEX]: I've changed the url. How does it work? Who puts the image on that url?

**Review by sonicaj — APPROVED** [review]
Thank you @Tostis for your contribution

**Pull request merged**